pages/apim/3.x/user-guide/publisher/backend-services.adoc

```diff
@@ -0,0 +1,134 @@
+= Load balancing, failover and health check
+:page-sidebar: apim_3_x_sidebar
+:page-permalink: apim/3.x/apim_publisherguide_backend_services.html
+:page-folder: apim/user-guide/publisher
+:page-layout: apim3x
+
+== Overview
+
+APIM includes a number of backend services for managing your APIs. This page explains how to configure the following services for your APIs:
+
+* Load balancing
+* Failover
+* Health check
+
+=== How to access the API backend service configuration
+
+To update the configuration for these services:
+
+. link:/apim/3.x/apim_quickstart_console_login.html[Log in to APIM Console^].
+. Click *APIs* and select an API.
+. Click *Proxy*.
++
+The backend service configuration options are in the *BACKEND SERVICES* section.
++
+image:apim/3.x/api-publisher-guide/backend-services/api-backend-services.png[]
+
+WARNING: You must redeploy your API with the *deploy the API* link after each new configuration.
+
+== Configure load balancing
+
+APIM Gateway includes a built-in load balancer, which you can enable and configure for your API endpoints according to your own requirements.
+
+[[endpoint-groups]]
+=== Endpoint groups
+
+You configure load balancing by creating a logical grouping of endpoints and associating a load balancing algorithm with them. Each endpoint URL in the group needs to represent an instance of the same API deployed on a different server.
```

Comment:
it's wrong to say that each endpoints needs to represent an instance of the same api. If you use the dynamic routing policy to select an endpoint, you override the loadbalancing algorithm and force gravitee to use the specific endpoint. If you select a group of endpoint in the policy, then you will use the load balancing mode configured in the selected group.

Reply:
Updated to make this an example of configuring endpoints rather than a firm statement